README.md

```diff
@@ -31,7 +33,9 @@ The subdomain registrar functions roughly as follows --- you give the registrar
 1. Set the `ADMIN_PASSWORD` environment var to strong password. 
 ex: 
 ```bash
-$ pwgen -c 64 1
+pwgen -c 64 1
```

Comment:
For this one, we should leave it as is since it shows the command *and* the output.